mobile/src/main/java/ch/epfl/sdp/mobile/state/LocalizedStrings.kt

```diff
@@ -1,20 +1,25 @@
 package ch.epfl.sdp.mobile.state
 
 import android.content.res.Configuration
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.CompositionLocalProvider
-import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.runtime.*
 import androidx.compose.ui.platform.LocalConfiguration
 import androidx.core.os.ConfigurationCompat
-import ch.epfl.sdp.mobile.ui.i18n.English
-import ch.epfl.sdp.mobile.ui.i18n.LocalizedStrings
+import ch.epfl.sdp.mobile.application.settings.SettingsFacade
+import ch.epfl.sdp.mobile.ui.i18n.*
+import java.util.Locale.FRENCH
+import java.util.Locale.GERMAN
 
 /**
  * An [androidx.compose.runtime.ProvidableCompositionLocal] which provides access to the
  * [LocalizedStrings] for this application.
  */
 val LocalLocalizedStrings = compositionLocalOf<LocalizedStrings> { English }
 
+class SWISSGERMAN {
+  companion object {
+    val language: String = "ch"
+  }
+}
```

Comment:
(discussed out of band : this should be an enum)

Reply:
thx